pkg/bootstrap/testdata/all_golden.json

```diff
@@ -232,6 +232,9 @@
           "suffix": "shadow_denied"
           },
           {
+          "safe_regex": {"google_re2":{}, "regex":"vhost.*.route.*"}
```

Comment:
`vhost.*\.route.*`

Do we need to escape the `.`?

Reply:
This is our first regex, is there perf cost associated here? @douglas-reid @kyessenov

Reply:
Good point about escaping. I forgot about that. Yes it needs to be escaped. @varks can you please fix it?

Reply:
@howardjohn thanks for catching this. Here's the fix: https://github.com/istio/istio/pull/40594